config/global.js

```diff
@@ -45,6 +45,7 @@ const {
 	HELPAREA_URL,
 	FEATURE_MATRIX_MESSENGER_ENABLED,
 	FEATURE_MESSENGER_SCHOOL_SETTINGS_VISIBLE,
+	FEATURE_MULTI_LOGIN_INSTANCES = true,
```

Comment:
Wouldn't it be better not to set a default here? So that developers who don't have the Nuxt client running can still have a smooth login flow?

Reply:
That's a good point. I will remove this default value.